packages/doenetml-prototype/src/renderers/doenet/point.tsx

```diff
@@ -12,6 +12,8 @@ import { serializedComponentsReviver } from "@doenet/utils";
 import { Action, PointProps } from "@doenet/doenetml-worker-rust";
 import { useAppDispatch } from "../../state/hooks";
 import { coreActions } from "../../state/redux-slices/core";
+//@ts-ignore
```

Comment:
This is a bit frustrating. Maybe we could re-export some typed versions of the functions that we're using?

Reply:
Sounds like a worthwhile goal. Seems overwhelming to me, so I could use some help getting started!

Reply:
I wasn't suggesting doing the whole thing. Just create and re-export the function you used in this file.

Reply:
Do you mean within DoenetML rather than in `math-expressions`? I could try that.

Reply:
I can only do that for a function that returns something simple like a number. The math expression object itself has a zillion attributes, so just describing that type would be non-trivial.

Reply:
done for this example